www/front_src/src/Extensions/Filter/Criterias/models.tsx

```diff
@@ -0,0 +1,85 @@
+import { SelectEntry } from '@centreon/ui';
+
+import { SortOrder } from '../../models';
+import {
+  labelInstalled,
+  labelModule,
+  labelNotInstalled,
+  labelOutdated,
+  labelUpdated,
+  labelType,
+  labelStatus,
+  labelWidget,
+} from '../../translatedLabels';
+
+export type CriteriaValue = Array<SelectEntry> | string | [string, SortOrder];
+
+export interface Criteria {
+  name: string;
+  value?: CriteriaValue;
+}
+
+const criteriaValueNameById = {
+  INSTALLED: labelInstalled,
+  MODULE: labelModule,
+  OUTDATED: labelOutdated,
+  UPDETED: labelUpdated,
+  WIDGET: labelWidget,
+  not_installed: labelNotInstalled,
```

Comment:
```suggestion
  NOT_INSTALLED: labelNotInstalled,
```

Reply:
eslint demande That i follow one of this snake_case, camelCase, PascalCase